README.md

```diff
@@ -1 +1,61 @@
-# Zend-Expressive-Bridge
\ No newline at end of file
+# Zend-Expressive-Bridge
+
+The easiest way to use PHP-DI with Zend Expressive, seems to:
+ 
+ - use the Dependency Injection Container Builder provided by this library using some default definition files
+ - add your own Dependency Definitions to the Container Builder
+ - use the Dependency Injection Container to fetch the Zend Expressive Application
+ - run the Application
+
+## In your ```./public/index.php```
+
+```php
+
+<?php
+// Delegate static file requests back to the PHP built-in WebServer
+if (php_sapi_name() === 'cli-server'
+    && is_file(__DIR__ . parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH))
+) {
+    return false;
+}
+
+require __DIR__ . '/../vendor/autoload.php';
+
+/* @var \Interop\Container\ContainerInterface $container */
+$container = require __DIR__ . '/../config/container.php';
+
+/* @var \Zend\Expressive\Application $app */
+$app = $container->get(\Zend\Expressive\Application::class);
+$app->run();
+
+```
+
+## In your ```./config/container.php```
+
+
+```php
+<?php
+/* @var \DI\ContainerBuilder $containerBuilder */
+$containerBuilder = require __DIR__ . '/../vendor/php-di/zend-expressive-bridge/config/containerBuilder.php';
+$inProduction = false; //You probably want to use an environment variable for this...
```

Comment:
Here is an idea: instead of requiring the `containerBuilder.php` file that is in `vendor/` there could be instead a class, e.g. `ExpressiveContainerBuilder`, that extends from `ContainerBuilder` and auto-add the configuration files of this repository.

The advantage I see is it benefits from autoloading so it avoids the long line of include that goes looking in `vendor/` (which is kind of unusual), and also the possibility to get rid of all the `if(class_exists('Zend\Expressive\Router\AuraRouter')){` in the config files => the `ExpressiveContainerBuilder` could have methods dedicated to include configuration for libraries.

Here is an example of what it could look like:

```php
// the $inProduction could be used to enable/disable Twig debug or other debug for example
$containerBuilder = new ExpressiveContainerBuilder($inProduction);
$containerBuilder->registerAuraRouter();
$containerBuilder->registerTwigRenderer();
// All methods of the container builder can still be used of course
$containerBuilder->addDefinitions(/* your own definition files */);
```

It's a bit less automatic than your current solution but relying on `class_exists` is risky IMO, it's not because a class is installed in `vendor/` that you want to use it in your application. And that way it's much more explicit/easier to understand what's happening.